### PR TITLE
consolidate databases

### PR DESCRIPTION
**Pull request by yufengwng**
## Context

Here are the changes for moving to a single database instance, and the scripts to run for consolidation.

## Checklist

- [x] I have labeled the type of changes involved using the `C-*` labels.
- [x] I have assessed potential risks and labeled using the `R-*` labels.
- [x] I have considered error handling and alerts, and added `L-*` labels as needed.
- [x] I have considered security and privacy, and added `I-*` labels as needed
- [x] I have analyzed my changes for stability, fault tolerance, graceful degradation, performance bottlenecks and written a brief summary in this PR.

## FUD Score

_Overall, how are you feeling about these changes?_

- [x] :relaxed: All good, business as usual!
- [ ] :sweat_smile: There might be some issues here
- [ ] :scream: I'm sweaty and nervous

## How do we demo these changes?

_How does one observe these changes in a deployed system? Note that **user visible** encompasses many personas--not just patients and study staff, but also ops duty, your fellow devs, compliance, etc._

- [ ] They are user-visible in dev as a regular user journey and require no additional instructions.
- [x] Getting dev into a state where this is user-visible requires some tech fiddling. I have documented these steps in the related ticket.
- [ ] Requires other features before it's human visible. I have documented the blocking issues in jira.
- [ ] I have no idea how to demo this. Please help me!

## Testing

- [ ] I have written automated positive tests
- [ ] I have written automated negative tests
- [ ] I have written zero automated tests but have poked around locally to verify proper functionality
- [x] The jira ticket has acceptance criteria and QA has the needed information to test changes

## Release

- [ ] These changes require no special release procedures--just code!
- [x] Releasing these changes requires special handling and I have documented the special procedures in the release plan document

### CODE REVIEW

File: pepper-apis/config/sqlproxy.env.ctmpl [hunk 1]
```diff
@@ -1,16 +1,17 @@
-{{with $environment := env "ENVIRONMENT"}}
+{{$environment := env "ENVIRONMENT"}}
+{{$version := env "VERSION"}}
+{{$response := vault (printf "secret/pepper/%s/%s/conf" $environment $version)}}
+{{$conf := $response.Data}}
+
 {{if eq $environment "dev"}}
-       CLOUDSQL_CONNECTION_LIST=broad-ddp-dev:us-central1:pepper-dev=tcp:0.0.0.0:3307,broad-ddp-dev:us-central1:housekeeping-dev1=tcp:0.0.0.0:3308
+    CLOUDSQL_CONNECTION_LIST=broad-ddp-dev:us-central1:{{$conf.dbInstanceId}}=tcp:0.0.0.0:3307
```
Comment (ssettipalli) [inline pepper-apis/config/sqlproxy.env.ctmpl]: I think we can substitute broad-ddp-xxxx from {{$conf.googleProjectId}}
This way we can get rid of All if statements and just have below line in the template 
CLOUDSQL_CONNECTION_LIST={{$conf.googleProjectId}}:us-central1:{{$conf.dbInstanceId}}=tcp:0.0.0.0:3307

File: scripts/consolidate-db/utf8mb4.tmpl.sql [hunk 2]
```diff
@@ -0,0 +1,93 @@
+-- This script helps migrate database to use the utf8mb4 character set and the
+-- utf8mb4_unicode_ci collation. utf8mb4 because it has full Unicode support,
+-- such as for emojis. utf8mb4_unicode_ci because it works with utf8mb4 and
+-- seems to give better sorts.
+--
+-- At a high-level, this script will conver the given database schema, all of
```
Comment (ssettipalli) [inline scripts/consolidate-db/utf8mb4.tmpl.sql]: typo: convert

**Review by ssettipalli — APPROVED** [review]
looks good to me.
Deploy should be coordinated with DSM because of any URL changes needed on DSM side.

**Comment by yufengwng** [comment]
please run tests

**Pull request merged**